spring-cloud-gateway-integration-tests/grpc/pom.xml

```diff
@@ -45,6 +45,36 @@
 			<artifactId>grpc-stub</artifactId>
 			<version>1.44.0</version>
 		</dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.dataformat</groupId>
+			<artifactId>jackson-dataformat-protobuf</artifactId>
+			<optional>true</optional>
+			<version>2.13.2</version>
+		</dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
+			<optional>true</optional>
+			<version>2.13.2</version>
```

Comment:
This version is managed by spring-boot via the jackson bom and should not be needed

Reply:
Removed, thanks!